wordpress-mobile/WordPress-Android
# Initial quick start tutorial prompts

Part of #7707

This PR adds functionality for showing Quick Start dialog snack bars (with icons) after the user picks tutorial from Quick Start screen.

To test:

1. Navigate to Quick Start activity from My Site fragment.
2. Click on each tutorial (Create Site is completed by default), and make sure that the Quick Start dialog snack bar with relevant text appears at MySite fragment.

Snackbar will not be recreated when orientation changes while it's still visible.

![screenshot-2018-07-10_22 01 37 981](https://user-images.githubusercontent.com/728822/42531531-e0205f2c-848c-11e8-8eb9-3b0b8273312c.png)
